libhb/handbrake/nvenc_common.h

```diff
@@ -18,20 +18,4 @@ int            hb_nvenc_av1_available();
 int            hb_check_nvenc_available();
 int            hb_check_nvdec_available();
 
-
-char*          hb_map_nvenc_preset_name (const char * preset);
```

Comment:
removing this will cause compilation to fail, since the function is still used in decavcodec.c/encavcodec.

Reply:
Whoops.